content/docs/hooks-state.md

```diff
@@ -187,6 +187,14 @@ In a function, we already have `setCount` and `count` as variables so we don't n
   </button>
 ```
 
+Alternatively, we can use the second form of setState() that accepts a function. That function will receive the previous state as the first argument `prevState`, which will always contain the modified state in the recent `setCount` call:
```

Comment:
I believe we have something similar here https://reactjs.org/docs/hooks-reference.html#functional-updates

Reply:
Good catch @harish-sethuraman. What about adding the link you provided as a hyperlink in the PR for reference purposes? Would that suffice?

Reply:
The page flow of the one you have edited seems to explain about states in class components vs state in functional components. we can better leave it as such in the hooks reference where it already is.